app/src/main/java/org/fossasia/openevent/general/order/OrdersUnderUserFragment.kt

```diff
@@ -136,9 +159,22 @@ class OrdersUnderUserFragment : Fragment(), BottomIconDoubleClick {
         super.onViewCreated(view, savedInstanceState)
 
         val recyclerViewClickListener = object : OrdersPagedListAdapter.OrderClickListener {
-            override fun onClick(eventID: Long, orderIdentifier: String, orderId: Long) {
-                findNavController(rootView).navigate(OrdersUnderUserFragmentDirections
-                    .actionOrderUserToOrderDetails(eventID, orderIdentifier, orderId))
+            override fun onClick(eventID: Long, orderIdentifier: String, orderId: Long, event: Event, order: Order) {
+                if (order.status.equals("completed") || order.status.equals("placed")) {
```

Comment:
Don't use equals

Reply:
ok

Reply:
@iamareebjamal Then how should I check the status is pending or completed.

Reply:
@liveHarshit What you have to say about this?

Reply:
How do you check for equality of 2 strings in kotlin?

Reply:
"==" ??

Reply:
Yes